### ISSUE TITLE
is the installation link dead?

### ISSUE THREAD

**Issue by shi27feng** [riscv/homebrew-riscv#49]
Hi,
I got the below error when I try to install homebrew-riscv
```shell
❯ ruby -e "$(curl -fsSL https://raw.github.com/Homebrew/homebrew/go/install)"
curl: (22) The requested URL returned an error: 404
```

**Comment by sbeamer** [comment]
Homebrew changed its install instructions for the new 3.0 release. I believe I fixed the README with 9fca29d.

**Comment by shi27feng** [comment]
Sorry, I should raise the issue to homebrew but not here. And thank you for the reply.
